server/src/pages/shot/editor.js

```diff
@@ -110,6 +374,14 @@ exports.Editor = class Editor extends React.Component {
       this.canvasContainer.removeEventListener("mousemove", this.draw);
```

Comment:
I know this is not new code, but I don't understand how `this.state.tool == 'none'` is ever possible.